{ 'stand': { 'left_arm': [],
             'left_leg': [ { 'duration': 10.02,
                             'name': u'stand_up_7_left',
                             'positions': [ 0.0,
                                            -0.062,
                                            -0.7369,
                                            1.1723,
                                            0.5921,
                                            -0.0622],
                             'starttime': 0.0}],
             'neck': [],
             'right_arm': [],
             'right_leg': [ { 'duration': 9.98,
                              'name': u'stand_up_7_right',
                              'positions': [ -0.0,
                                             0.0618,
                                             0.7369,
                                             -1.1723,
                                             -0.5921,
                                             0.0616],
                              'starttime': 0.0}],
             'torso': []}}
